docs/Earn/liquid-staking/matic/matic-liquid-staking-mechanics.md

```diff
@@ -0,0 +1,140 @@
+
+Currently, the user pays no fees while unstaking.
+
+The user must also count in the gas price for outgoing transactions.
+
+
+## Rewards
+User's aMATICb balance grows daily. It can be desribed as `daily rewards / total share * user's share`.
+
+Validators receive rewards on the regular basis at the end of each epoch.
+
+The APY is calculated from the validators' rewards.
+
+## Validators
+
+There is a single [Ankr validator on Polygon](https://wallet.polygon.technology/staking/validators/31). 
+
+
+## Smart contracts
+Smart contracts and addresses involved in MATIC Liquid Staking are:
+
+* [Matic](https://etherscan.io/address/0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0) — contract on Ethereum that the user calls to approve the chosen MATIC amount to be sent to the PolygonPool
+
+* [PolygonPool](https://etherscan.io/address/0xCfD4B4Bc15C8bF0Fd820B0D4558c725727B3ce89) — contract on Ethereum where the user sends their initial staking request.
+
+* [aMATICb](https://etherscan.io/address/0x99534Ef705Df1FFf4e4bD7bbaAF9b0dFf038EbFe) — contract on Ethereum that mints aMATICb tokens for the user in 1:1 ratio with the staked amount.
+
+Smart contracts code lives by the links above, at the **Contract** tab on the page.
+
+
+## Staking workflow
+The workflow is both user and Ankr-driven. The user part is Steps 1–2, the Ankr part is Steps 3–7.
+
+User calls `Matic::approve(address spender, uint256 amountToSpend)` to approve the chosen MATIC amount to be sent to the `PolygonPool` smart contract.
+
+User sends a request to Ankr `PolygonPool` smart contract on Ethereum, calling `PolygonPool::stake(uint256 amount)`. The request is sent via the Ankr Earn Dashboard and Metamask or another wallet connected to Ethereum.
+
+Upon receiving the stake, the `Polygon Pool` smart contract calculates the equivalent amount of aMATICb and mints them to the user’s address in the stake transaction.
+
+`Polygon Pool` emits a `StakePending(msg.sender, amount)` event.
+
+Ankr Earn Dashboard allows the user to register (import) the received aMATICb tokens to Metamask or another wallet connected to Ethereum.
+
+Ankr backend collects the MATICs staked by users, transferring them from the smart contract to the address controlled by the platform, and makes a validator stake on the Ankr Matic Validator node if the conditions below are met.
+
+Ankr Earn Dashboard can call `aMATICb::balanceOf(address account)` to check the user’s current aMATICb balance.
+
+ANK backend will monitor blockchain events coming from the `PolygonPool` smart contract. Upon a new staking event, the backend:
+
+1. Checks if the newly available MATIC number (*N*) is >= to the minimum threshold. If yes, the available MATICs are sent from the `PolygonPool`  smart contract to the address managed by Ankr. Else, the backend waits for the next stake event from `PolygonPool`. *N* is a config parameter.
+
+2. When MATIC tokens become available at the Ankr address, staking transaction is issued.
```

Comment:
At the point when MATIC tokens are available at the Ankr address, the..

Reply:
thx